Packaging.props

```diff
@@ -1,5 +1,6 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <LicenseUrl>https://github.com/dotnet/corefx/blob/master/LICENSE</LicenseUrl>
```

Comment:
@tarekgh @weshaggard why does this point to corefx when the LICENSE file in this repo already has the MIT license ?